packages/adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -3,12 +3,31 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:masonry_grid/masonry_grid.dart';
+
 import 'adaptive_layout.dart';
 import 'breakpoints.dart';
 import 'slot_layout.dart';
 
-/// Implements the basic visual layout structure for [Material Design 3](https://m3.material.io/foundations/adaptive-design/overview) that
-/// adapts to a variety of screens.
+/// Gutter value between different parts of the body slot depending on
+/// material 3 design spec.
+const double materialGutterValue = 8;
+
+/// margin value of the compact breakpoint layout according to the material
+/// design 3 spec
+const double materialCompactMinMargin = 8;
```

Comment:
Also by convention of the flutter/flutter repo, use prefix `k` to mark constant variables.
```suggestion
const double kMaterialCompactMinMargin = 8;
```